public/controllers/agent/components/register-agent.js

```diff
@@ -473,7 +485,14 @@ export const RegisterAgent = withErrorBoundary(
 
       const guide = (
         <div>
-          {this.state.selectedOS && (
+          {(!this.state.needsPassword && this.state.wazuhPassword=== "") ? (
+            <EuiCallOut
+              color="warning"
+              title={<> This section could not be displayed because you do not have permission to get access to the registration service. </>}
```

Comment:
why `registration service`? maybe `authorization coniguration` (?)

Reply:
**reply**:
Wazuh documentation about the configuration that is requested:
https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/auth.html#auth

![image](https://user-images.githubusercontent.com/34042064/124455046-6cc46380-dd89-11eb-8eec-45bc5d89d63e.png)

Reply:
I think to have a relationship with the documentation. we should leave it like this